ibis/backends/tests/test_export.py

```diff
@@ -311,6 +313,7 @@ def test_table_to_csv(tmp_path, backend, awards_players):
     reason="cannot inline WriteOptions objects",
     raises=DuckDBParserException,
 )
+@pytest.mark.never(["pyspark"], reason="backend writes a dir", raises=IsADirectoryError)
```

Comment:
Does this result in effectively no testing for PySpark `to_csv`?

Reply:
No; I added equivalent tests in ibis/backends/pyspark/tests/test_export.py.